### PR TITLE
Add clevis_to_str function

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #958

### CODE REVIEW

**Comment by bgurney-rh** [comment]
Example output:

tang (redacted):
```
Clevis Configuration: tang   thp: z_XXXXXXXXXXXXXX_XXX_XXXXXXXXXXXXXX-XXXXXXX url: http://127.0.0.1
```

TPM 2.0 (emulated):
```
Clevis Configuration: tpm2   
```

**Review by mulkieran — CHANGES_REQUESTED** [review]
Requesting one significant change.

**Comment by mulkieran** [comment]
@bgurney-rh Note that we do not worry about coverage until the end; so it's ok if the coverage task keeps failing.

File: src/stratis_cli/_actions/_utils.py [hunk 1]
```diff
@@ -44,7 +44,10 @@ def __init__(self, pin, config):
         :param dict config: the JSON config corresponding to the pin
         """
         self.pin = pin
-        self.config = config
+        self.config = " ".join(f"{key}: {value}" for key, value in config.items())
```
[the remark this answers is not in the document]
Comment (bgurney-rh) [reply src/stratis_cli/_actions/_utils.py]: Resolved.

**Comment by mulkieran** [comment]
@drckeefe Do you find the output format acceptable?

**Review by drckeefe — APPROVED** [review]

**Comment by mulkieran** [comment]
@bgurney-rh Put ```no cover``` pragma's on both your newly introduced methods, like:

```
def clevis_to_str(clevis_dbus_object): # pragma: no cover
```

and run ```make fmt``` to get that formatted correctly.

It's going to be fairly hard to add tests, for fairly limited benefit, so just not trying to cover them is best, for now.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Just add the ```no cover``` pragmas, squash it all into a single commit, and we can merge.

**Comment by bgurney-rh** [comment]
All tests passed.  Squashing into a single commit.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**